WordPress/Jetpack/Classes/System/JetpackWindowManager.swift

```diff
@@ -43,6 +43,6 @@ class JetpackWindowManager: WindowManager {
 
     // TODO: Add logic in here to trigger migration UI if needed
     private var shouldShowMigrationUI: Bool {
-        false
+        true
```

Comment:
I am not sure we want to change this just yet, because the value of this flag depends on other conditions

Reply:
Yes of course, I'll revert this back to `false`.